### PR TITLE
Release/172.0.0

### PR DESCRIPTION
**Pull request by MajorLift**
## Explanation

Contains patch releases of the following commit for `base-controller`, `permission-controller`, and all of their upstream deps/peerDeps:

- https://github.com/MetaMask/core/pull/4516

In the packages downstream of the release candidates, I'm using the `"intentionally-skip"` option, and only bumping the release candidates to their post-release versions in package.json. This way, our yarn constraints are satisfied in the main branch.

The version bumps will stay as unreleased changes which we can deploy in a subsequent release PR. Since the package versions are unchanged, the updates to the dependencies should be treated the same as any unreleased version bump, not disrupting the `create-release-branch` script or `publish-release` action in this PR or the upcoming release PR.

To prevent any issues with inconsistency, I plan to release all of these downstream packages simultaneously in the immediately subsequent release (173.0.0).

### Release order roadmap

- [core release 172.0.0] `{base,permission}-controller` 
- [snaps release 56.0.0] `snaps-{sdk,utils,rpc-methods}`
- [core release 173.0.0] all core controllers (except `snaps-controllers` consumers)
- [snaps release 57.0.0] `snaps-controllers`
- [core release 174.0.0] `{accounts,chain,profile-sync}-controller`

## References

- See https://github.com/MetaMask/core/pull/3645
- See https://github.com/MetaMask/snaps/pull/2445

## Checklist

- [x] I've updated the test suite for new or updated code as appropriate
- [x] I've updated documentation (JSDoc, Markdown, etc.) for new or updated code as appropriate
- [x] I've highlighted breaking changes using the "BREAKING" category above as appropriate

### CODE REVIEW

**Comment by MajorLift** [comment]
@mcmire This is an attempt at releasing `base-controller` without also releasing all of its downstream packages. Do you think this might be safe to do if we can assume that the downstream packages will be released in 173.0.0?

File: packages/approval-controller/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [7.0.1]
+
+### Changed
+
+- Bump `@metamask/rpc-errors` from `6.2.1` to `^6.3.1` ([#4516](https://github.com/MetaMask/core/pull/4516))
+- Bump `@metamask/utils` from `^8.3.0` to `^9.0.0` ([#4516](https://github.com/MetaMask/core/pull/4516))
```
Comment (mcmire) [inline packages/approval-controller/CHANGELOG.md:15]: Also:

``` suggestion
- Bump `@metamask/utils` from `^8.3.0` to `^9.0.0` ([#4516](https://github.com/MetaMask/core/pull/4516))
- Bump `@metamask/base-controller` to `^6.0.1` ([#4517](https://github.com/MetaMask/core/pull/4517))

File: packages/permission-controller/CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.0.1]
+
+### Changed
+
+- Bump `@metamask/rpc-errors` from `6.2.1` to `^6.3.1` ([#4516](https://github.com/MetaMask/core/pull/4516))
+- Bump `@metamask/utils` from `^8.3.0` to `^9.0.0` ([#4516](https://github.com/MetaMask/core/pull/4516))
```
Comment (mcmire) [inline packages/permission-controller/CHANGELOG.md:15]: Also:

``` suggestion
- Bump `@metamask/utils` from `^8.3.0` to `^9.0.0` ([#4516](https://github.com/MetaMask/core/pull/4516))
- Bump `@metamask/base-controller` to `^9.0.1` ([#4517](https://github.com/MetaMask/core/pull/4517))
- Bump `@metamask/controller-utils` to `^11.0.1` ([#4517](https://github.com/MetaMask/core/pull/4517))
- Bump `@metamask/json-rpc-engine` to `^9.0.1` ([#4517](https://github.com/MetaMask/core/pull/4517))

**Review by mcmire — COMMENTED** [review]
Just a few suggestions.

File: packages/permission-controller/CHANGELOG.md (hunk 2 above)
Comment (MajorLift) [reply packages/permission-controller/CHANGELOG.md:15]: Ah missed these. Good catch! Thanks :)

**Comment by mcmire** [comment]
@MajorLift To answer your question from above, yes, it's okay to release upstream packages before downstream packages, as long as the downstream packages only have direct dependencies on the upstream ones. It's when we have peer dependencies that things get a bit tricky, but that doesn't seem to be the case here. So I believe this PR should be okay.

**Review by mcmire — APPROVED** [review]
LGTM!

**Comment by MajorLift** [comment]
@mcmire Awesome, makes sense. Thanks for clearing that up!

**Review by Mrtenz — APPROVED** [review]

**Pull request merged**